web_app/ingest_page/ingest_process.py

```diff
@@ -30,16 +30,31 @@ def __init__(self, master, classifier, end_process=None, **kwargs):
         self.reference_table = DataFrameWidget(self, df_reference_table, self.interest_row, 2)
         self.reference_table.pack(fill='both', expand=True)
 
-        self.row_entry.add_entry_row_at(*self.classifier.get_entry_prerequisites_for_manual_entry(self.interest_row))
+        self.last_entry_row = self.classifier.get_entry_prerequisites_for_manual_entry(self.interest_row)
+        self.row_entry.add_entry_row_at(*self.last_entry_row)
 
     def data_entered(self, data):
         if data is not None:
             self.classifier.process_incoming_input(data)
 
-        self.interest_row += 1
-        self.row_entry.add_entry_row_at(*self.classifier.get_entry_prerequisites_for_manual_entry(self.interest_row))
+        if self.reference_table.scroll(1):
```

Comment:
this if condition is unclear please specify with a comment or rename the function

Reply:
I am guessing this is handling the end of table case in which case you should consider re naming the method to "try_scroll_to" or "try_scroll"